### PR TITLE
Lakhveer/dialog bot comment fix

### PR DESCRIPTION
**Pull request by lakhveerk**
For more information about how to contribute to this repo, visit [this page](https://github.com/OfficeDev/microsoft-teams-library-js/blob/main/CONTRIBUTING.md).

## Description
Fix dialog.url.bot documentation.

No Unit Tests or e2e tests are needed since this is not a code change

### CODE REVIEW

File: packages/teams-js/src/public/dialog.ts [hunk 1]
```diff
@@ -249,7 +249,7 @@ export namespace dialog {
      */
     export namespace bot {
       /**
-       * Allows an app to open the dialog module using bot.
+       * Allows an app to open the dialog module that sends the submit data to the bot.
```
Comment (TrevorJoelHarris) [inline packages/teams-js/src/public/dialog.ts:252]:
```suggestion
       * Allows an app to open the dialog module that sends the submit data to a bot.
```

**Review by TrevorJoelHarris — APPROVED** [review]

File: packages/teams-js/src/public/dialog.ts (hunk 1 above)
Comment (AE-MS) [reply packages/teams-js/src/public/dialog.ts:252]: Also consider "Allows an app to open a dialog that sends submitted data to a bot"

**Review by AE-MS — CHANGES_REQUESTED** [review]
:clock1:

File: packages/teams-js/src/public/dialog.ts (hunk 1 above)
Comment (lakhveerk) [reply packages/teams-js/src/public/dialog.ts]: Thank you for the suggestions! ❤️

**Review by AE-MS — APPROVED** [review]

**Review by shrshindeMSFT — APPROVED** [review]

**Pull request merged**